src/storage/src/hummock/cache.rs

```diff
@@ -617,11 +661,7 @@ impl<K: LruKey, T: LruValue> LruCache<K, T> {
     }
 
     fn shard(&self, hash: u64) -> usize {
-        if self.num_shard_bits > 0 {
-            (hash >> (64 - self.num_shard_bits)) as usize
-        } else {
-            0
-        }
+        hash as usize % self.shards.len()
```

Comment:
Why use mod instead of right shift?

Reply:
They are similar

Reply:
Seems that mod is more expensive than a right shift. 

And if we use the original right shift, we use the high bits of the hash value, which can be separated from the bits we use for hash table, and the load can probably be more dispersed.